notebooks/MAST/Kepler/kepler_interactively_inspecting_target_pixel_files_and_light_curves/requirements.txt

```diff
@@ -0,0 +1 @@
+lightkurve>=2.0b2
```

Comment:
Does lightkurve automatically pull in all the Bokeh stuff? I don't see `bokeh` defined here.

Reply:
I confirm `bokeh` is a dependency of `lightkurve`.